docs/programs/index.md

```diff
@@ -61,7 +61,7 @@ manner.
 Once the entire program has been sent to the blockchain, a final transaction is
 sent to write all of the buffered byte-code to the program's data account. This
 either mark the new program as
-[`executable`](/docs/core/accounts.md#executable), or complete the process to
+[`executable`](/docs/terminology#onchain-program), or complete the process to
```

Comment:
this seems like a useless link to change too. I vote to just remove the link

Reply:
removed